### PR TITLE
Update obib.md contact info

### PR DESCRIPTION
**Pull request by jmwhorton**

### CODE REVIEW

File: ontology/obib.md [hunk 1]
```diff
@@ -3,10 +3,9 @@ layout: ontology_detail
 id: obib
 title: Ontology for Biobanking
 contact:
-  email: jiezheng@pennmedicine.upenn.edu
-  github: zhengj2007
-  label: Jie Zheng
-  orcid: 0000-0002-2999-0103
+  email: jmwhorton@uams.edu
+  github: jmwhorton
+  label: Justin Whorton
```
Comment (cthoyt) [inline ontology/obib.md:8]: @jmwhorton needs a valid orcid

Comment (jmwhorton) [reply ontology/obib.md:8]: Updated and added orcid.  (Apologies, I didn't have one yet.)

**Comment by cthoyt** [comment]
@zhengj2007 can you confirm you want to switch the primary contact for OBIB?

**Comment by zhengj2007** [comment]
> @zhengj2007 can you confirm you want to switch the primary contact for OBIB?

Yes. Please merge the changes. Thanks!

**Review by cthoyt — APPROVED** [review]

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**